magellan-sample/src/main/java/com/wealthfront/magellan/sample/IntroStep.kt

```diff
@@ -1,16 +1,29 @@
 package com.wealthfront.magellan.sample
 
-import android.content.Context
-import com.wealthfront.magellan.core.Step
-import com.wealthfront.magellan.sample.databinding.IntroBinding
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Devices
+import androidx.compose.ui.tooling.preview.Preview
+import com.example.magellan.compose.ComposeStep
 
 internal class IntroStep(
   private val goToLearnMore: () -> Unit
-) : Step<IntroBinding>(IntroBinding::inflate) {
+) : ComposeStep() {
 
-  override fun onShow(context: Context, binding: IntroBinding) {
-    binding.learnMore.setOnClickListener {
-      goToLearnMore()
-    }
+  @Composable
+  override fun Compose() = IntroStepContent(goToLearnMore)
+}
+
+@Composable
+internal fun IntroStepContent(goToLearnMore: () -> Unit) {
+  Button(onClick = goToLearnMore) {
+    Text("Go to learn more")
   }
 }
```

Comment:
On second thought, this probably belongs inside the `Step`. That way, it can access the data inside the `Step`, which we can fake for the previews.

```kotlin
class MyStep : ComposeStep() {
  val myDataFlow = StateFlow<MyData>()

  @Composable
  override fun Compose() {
    val myData = myDataFlow.collectAsState()
    // Use myData here
  }
}
```